core-ui/src/components/Extensibility/components/Widget.js

```diff
@@ -72,6 +73,14 @@ export function Widget({ structure, value, inlineRenderer, ...props }) {
   } = useRelationsContext();
 
   let childValue;
+
+  if (!structure || typeof structure !== 'object') {
+    throwConfigError(t('extensibility.not-an-object'), structure);
+  }
+  if (!structure.path && !structure.children && !Array.isArray(structure)) {
```

Comment:
can we add a more general handler for the path - it not only cannot be empty, but it also cannot be a boolean for example.